<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelChooseWindow</class>
 <widget class="QMainWindow" name="LevelChooseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Level</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2" alignment="Qt::AlignTop">
     <widget class="QComboBox" name="comboBoxLevel"/>
    </item>
    <item row="1" column="0" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="StartButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="HelpButton">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
